python/GafferImageUI/CatalogueUI.py

```diff
@@ -190,6 +190,44 @@ def value( self, image, catalogue ) :
 registerColumn( "Frame", ContextVariableColumn( "Frame", "frame" ) )
 registerColumn( "Description", ImageMetadataColumn( "Description", "ImageDescription" ) )
 
+# Image properties
+
+def __resolutionColumnValueProvider( image, catalogue ) :
+
+	format_ = catalogue["out"].format()
+	return "%d x %d" % ( format_.width(), format_.height() )
+
+def __formatBox( box ) :
+
+	return "(%d %d) - (%d, %d)" % ( box.min().x, box.min().y, box.max().x, box.max().y )
+
+registerColumn(
+	"Image/Resolution",
+	SimpleColumn( "Resolution", __resolutionColumnValueProvider )
+)
+registerColumn(
+	"Image/Channels",
+	SimpleColumn( "Channels", lambda _, c : ", ".join( c["out"].channelNames() ) )
+)
+registerColumn(
+	"Image/Pixel Type",
+	SimpleColumn( "Pixel Type", lambda _, c : "Deep" if c["out"].deep() else "Flat" )
```

Comment:
Pixel Type -> Image Type perhaps? When I saw "Pixel Type" I thought int/half/float, even though I know Gaffer only deals in floats.